demos/kitchen_sink/main.py

```diff
@@ -15,6 +15,7 @@
 from libs.baseclass.list_items import (  # NOQA: F401
     KitchenSinkOneLineLeftIconItem,
 )
+from kivy.clock import Clock
```

Comment:
Please follow the standards for importing modules: first, modules of the standard library are imported, then modules and external packages, and at the end - your own modules and packages.

Reply:
bro i had used `isort` for sorting modules

Reply:
As you can see, `isort` is not working correctly.

Reply:
> bro i had used `isort` for sorting modules

mine problem sorry 🙏

Reply:
```
Folder Tree
KivyMD (Cloned)
|____demos
|____docs
|____kivymd
```
There are no lint errors in PR but `isort` looks the kivymd folder as a local module
```
Folder Tree
KivyMD (Cloned)
|____demos
|____docs
```
By deleting the kivymd folder the `isort` works correctly as u expected but lint error appears in PR

Reply:
I do not understand what the problem is, importing modules according to `PEP8`?

![ezgif-4-f1682d9a805e](https://user-images.githubusercontent.com/16930280/98782290-c4801a00-2408-11eb-8b80-c2f23447e9e0.gif)